verifier/src/channel.rs

```diff
@@ -35,6 +35,9 @@ pub struct VerifierChannel<
     // constraint queries
     constraint_commitment: H::Digest,
     constraint_queries: Option<ConstraintQueries<E, H, V>>,
+    partition_size_main: usize,
+    partition_size_aux: usize,
+    partition_size_constraint: usize,
```

Comment:
Let's add a comment line above these as we have for other variables.

An alternative could be to create a helper structure to hold these.

Reply:
Add a comment